description/active/universal/15 Credits.md

```diff
@@ -91,31 +91,32 @@ Firstly, a huge thanks to our former staff, developers, and anyone else who has
 - Beesafree
 - BloodlessNS
 - ChauzuVGC
-- ScruffyKip
 - KoolboymanAdam
-- LightningXCE
 - luckytyphlosion
+- Luna
 - m4_used_rollout
 - Made_s
+- Monstarules
 - NieDzejkob
 - PikalaxALT
 - PokemonGod777
 - raining_blood
 - Rangi
+- ScruffyKip
 - StarsMmd
 - Tustin2121
 - Umericchi
 *****
-## Rom Testers
+## ROM Testers
 - Cosmog
 - Deadinsky
 - Faithfulforce
-- ScruffyKip
-- LightningXCE
```

Comment:
Why?

Reply:
Discussed in PM about Kipp, Lightning was accidentally removed as per prior comment.